gax/src/createApiCall.ts

```diff
@@ -72,9 +76,22 @@ export function createApiCall(
     callOptions?: CallOptions,
     callback?: APICallback
   ) => {
-    const thisSettings = settings.merge(callOptions);
-
     let currentApiCaller = apiCaller;
+
+    let thisSettings: CallSettings;
+    if (currentApiCaller instanceof StreamingApiCaller) {
+      const gaxStreamingRetries =
+        currentApiCaller.descriptor?.gaxStreamingRetries;
```

Comment:
If you add `?? false`, it will be a real `boolean` (without possibly being undefined).